engine/installation/linux/ubuntulinux.md

```diff
@@ -116,365 +119,494 @@ packages from the Docker repository:
+* [Create a `docker` group](ubuntulinux.md#create-a-docker-group)
 * [Adjust memory and swap accounting](ubuntulinux.md#adjust-memory-and-swap-accounting)
 * [Enable UFW forwarding](ubuntulinux.md#enable-ufw-forwarding)
 * [Configure a DNS server for use by Docker](ubuntulinux.md#configure-a-dns-server-for-use-by-docker)
 * [Configure Docker to start on boot](ubuntulinux.md#configure-docker-to-start-on-boot)
 
-### Create a Docker group
+### Create a `docker` group
 
 The `docker` daemon binds to a Unix socket instead of a TCP port. By default
-that Unix socket is owned by the user `root` and other users can access it with
-`sudo`. For this reason, `docker` daemon always runs as the `root` user.
+that Unix socket is owned by the user `root` and other users can only access it
+using `sudo`. The `docker` daemon always runs as the `root` user.
 
-To avoid having to use `sudo` when you use the `docker` command, create a Unix
+If you don't want to use `sudo` when you use the `docker` command, create a Unix
 group called `docker` and add users to it. When the `docker` daemon starts, it
 makes the ownership of the Unix socket read/writable by the `docker` group.
 
->**Warning**: The `docker` group is equivalent to the `root` user; For details
->on how this impacts security in your system, see [*Docker Daemon Attack
->Surface*](../../security/security.md#docker-daemon-attack-surface) for details.
+>**Warning**: The `docker` group is equivalent to the `root` user. For details
+on how this impacts security in your system, see [*Docker Daemon Attack
+Surface*](../../security/security.md#docker-daemon-attack-surface) for details.
 
 To create the `docker` group and add your user:
 
-1. Log into Ubuntu as a user with `sudo` privileges.
+1.  Log into Ubuntu as a user with `sudo` privileges.
 
-2. Create the `docker` group.
+2.  Create the `docker` group.
+    ```bash
+    $ sudo groupadd docker
+    ```
 
-        $ sudo groupadd docker
+3.  Add your user to `docker` group.
 
-3. Add your user to `docker` group.
+    ```bash
+    $ sudo usermod -aG docker $USER
+    ```
 
-        $ sudo usermod -aG docker $USER
+4.  Log out and log back in so that your group membership is re-evaluated.
 
-4. Log out and log back in.
+5.  Verify that you can `docker` commands without `sudo`.
 
-    This ensures your user is running with the correct permissions.
+    ```bash
+    $ docker run hello-world
+    ```
 
-5. Verify your work by running `docker` without `sudo`.
+	  If this fails, you will see an error:
 
-        $ docker run hello-world
+    ```none
+		Cannot connect to the Docker daemon. Is 'docker daemon' running on this host?
+    ```
 
-	If this fails with a message similar to this:
+	  Check that the `DOCKER_HOST` environment variable is not set for your shell.
+	  If it is, unset it.
 
-		Cannot connect to the Docker daemon. Is 'docker daemon' running on this host?
+    ```bash
+    $ unset DOCKER_HOST
```

Comment:
Now that we show how to unset, may be worth showing how to find if it's set, something like:

```bash
$ env | grep DOCKER_HOST
```